python/mxnet/optimizer.py

```diff
@@ -174,14 +174,24 @@ def set_lr_scale(self, args_lrscale): # pylint: disable=unused-argument
         raise DeprecationWarning
 
     def set_lr_mult(self, args_lr_mult):
-        """Set individual learning rate for each weight.
+        """Set an individual learning rate multiplier for each parameter.
+
+        If you specify a learning rate multiplier for a parameter, then
+        the learning rate for the parameter will be set as the product of
+        the global learning rate `self.lr` and its multiplier.
+
+        .. note:: The default learning rate multiplier of a `Symbol`
```

Comment:
you can only set it for variables and its lr_mult (without the underscore) in the interface